app/models/ncr/work_order.rb

```diff
@@ -120,11 +122,21 @@ def current_approver_email_address
         self.currently_awaiting_approvers.first.email_address
       elsif self.approving_official
         self.approving_official.email_address
+      elsif self.emergency and self.approvers.empty?
+        EMERGENCY_APPROVER_EMAIL
```

Comment:
`nil` is somewhat ambiguous as a return value. Does it mean "there wasn't an email address" or "you were wrong to ask in the first place" or "I couldn't find what you wanted" or "this is a specific case where nil is important"?

I made it a constant though `EMERGENCY_APPROVER_EMAIL` so that we can compare and/or change it easily. If everyone wants the constant value to be `nil` I won't object. Just wanted it to be clear when reading that it was a special case.